docs/getting-started/beginner-guide.md

```diff
@@ -1,85 +1,102 @@
 
-|           | dcrd | dcrwallet | dcrctl | Decrediton  |
-| ---------:|:----:|:---------:|:------:|:-----------:|
-| Windows   | X    | X         | X      | X           |
-| macOS     | X    | X         | X      | X           |
-| Linux     | X    | X         | X      | X           |
-| Other UNIX| X    | X         | X      |             |
+* [Obtaining DCR](/getting-started/obtaining-dcr.md)
+* [Using the Block Explorer](/getting-started/using-the-block-explorer.md)
+* [Governance](/governance/governance/)
+* [Ticket buying guide](/mining/proof-of-stake.md)
+* [Politeia](/governance/politeia.md)
+* [Voting on consensus rules changes](/getting-started/user-guides/agenda-voting.md)
+* [Using Testnet](/getting-started/using-testnet.md)
 
-"Other UNIX" currently includes various *BSDs and Solaris/illumos.
 
-Note: You might soon notice one of the many differences between Decred and other 
-popular cryptocurrencies: the wallet daemon and node daemon are separate.
-A lot of other coins run these functions together in a single daemon.
-For those who choose to use the command line interfaces, this means you must
-run `dcrd` for full node functionality, and `dcrwallet` to store your DCR,
-create transactions and participate in Proof-of-Stake mining/voting.
+## Information
 
----
+**Network Stats**
 
-## Installation Guides 
+* https://stats.decred.org/
 
-To get started, choose an option for installations available for your operating system below:
+**Block Explorers**
 
-* [Command Line Suite **via dcrinstall**](/getting-started/user-guides/cli-installation.md) - NOTE: The `dcrinstall` method is the fastest and easiest way to get a command-line node and wallet running. It is recommended and the command-line path setup guides below will assume you've used this method.
-* [Decrediton](/getting-started/user-guides/decrediton-setup.md)
+* https://explorer.dcrdata.org/
+* https://mainnet.decred.org/
 
----
+**News/Articles**
 
-**NOTE:** All of the following guides can be found in the navigation menu with the same names.
+* [Decred journal](https://xaur.github.io/decred-news/) - comprehensive monthly newsletter maintained by some community members
+* [Decred.org blog](https://blog.decred.org)
+* [Decred Medium](https://medium.com/decred) (curated content from community members)
+* [Press coverage/releases](https://decred.org/press/)
+* [The Decred Digest](https://thedecreddigest.com/) - community blog
 
-## Decrediton Path 
+**Video**
 
-The following guide will help you load Decrediton:
+* [Decred Youtube channel](https://www.youtube.com/decredchannel)
 
-* [Decrediton Setup](/getting-started/user-guides/decrediton-setup.md)
-* [Using Decrediton](/getting-started/user-guides/using-decrediton.md)
+## Community
 
-## Command-Line Path 
+In addition to the [Politeia proposals site](www.proposals.decred.org), the Decred community can be found on a [variety of platforms](https://decred.org/community/).
```

Comment:
Politeia link does not work. Should be:
`[Politeia proposals site](https://proposals.decred.org)`
instead of
`[Politeia proposals site](www.proposals.decred.org)`